index.js

```diff
@@ -174,8 +174,10 @@ async function terraform(args) {
     tf_fmt = status_success;
   }
 
+  const terraformTargetArgs = terraformTargets.map((target) => `-target '${target}'`);
```

Comment:
Why not do the mapping on line 46 instead? Or at least move this close to that. Now it feels a bit lost in the wilderness here :wink: